lib/gems/pending/appliance_console/external_httpd_configuration.rb

```diff
@@ -15,6 +15,7 @@ module ExternalHttpdConfiguration
       SSSD_CONFIG          = "/etc/sssd/sssd.conf".freeze
       PAM_CONFIG           = "/etc/pam.d/httpd-auth".freeze
       HTTP_KEYTAB          = "/etc/http.keytab".freeze
+      HTTP_CONFIG_FILE     = "/etc/httpd/conf.d/manageiq-redirects-ws".freeze
```

Comment:
@abellotti and @carbonin I could check if the http service is running but that could be an overly restrictive use case. It would be possible for a user to configure external auth if the appliance had already been configured but httpd could have been stopped.

But perhaps requiring httpd to be running would not be unreasonable and it would allow us to avoid the concern @carbonin raised.

@abellotti and @carbonin Thoughts?